pkg/misc/depversion/depversion_test.go

```diff
@@ -485,16 +485,3 @@ func Test_DoesRangeInclude(t *testing.T) {
 		})
 	}
 }
-
-func Test_DoesRangeInclude_Errors(t *testing.T) {
```

Comment:
can we have a bit more descriptive PR name / commit names, else lgtm

Reply:
Sounds good, done.